### PR TITLE
Add some table printer

### PR DESCRIPTION
**Pull request by wanghaoran1988**
**What this PR does / why we need it**:
Add table printer for Service resource

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: 
related to https://github.com/kubernetes/kubernetes/issues/48820

**Special notes for your reviewer**:

**Release note**:

```
None
```

### CODE REVIEW

**Comment by wanghaoran1988** [comment]
@kubernetes/sig-cli-pr-reviews

**Comment by wanghaoran1988** [comment]
@fabianofranz Would  you please help review this?

File: pkg/printers/internalversion/printers.go [hunk 1]
```diff
@@ -196,8 +194,19 @@ func AddHandlers(h printers.PrintHandler) {
 	h.TableHandler(cronJobColumnDefinitions, printCronJob)
 	h.TableHandler(cronJobColumnDefinitions, printCronJobList)
 
-	h.Handler(serviceColumns, serviceWideColumns, printService)
-	h.Handler(serviceColumns, serviceWideColumns, printServiceList)
+	serviceColumnsDefinitions := []metav1alpha1.TableColumnDefinition{
```
Comment (fabianofranz) [inline pkg/printers/internalversion/printers.go]: nit, `serviceColumnDefinitions`

**Comment by fabianofranz** [comment]
@wanghaoran1988 looking good. :+1: just fix the nit and this is good to go.

**Comment by wanghaoran1988** [comment]
/retest
@fabianofranz Add some more table printer

**Comment by fabianofranz** [comment]
Awesome @wanghaoran1988 thanks!

/lgtm
/approve no-issue

**Pull request merged**